app/models/post.rb

```diff
@@ -0,0 +1,27 @@
+class Post < ApplicationRecord
+  # Establishes a belongs_to association with the User model, using author_id as the foreign key.
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
+  # Establishes has_many associations with the Comment and Like models.
+  has_many :comments
+  has_many :likes
+
+  # Validations for the Post model
+  validates :title, presence: true
+  validates :text, presence: true
```

Comment:
- [ ] As mentioned by the previous :point_up: reviewer please add all validations mentioned by microverse but for now you just add the title and text not be blank so add the following validations as well :slightly_smiling_face: 

- The `title` must not exceed 250 characters.
- `CommentsCounter` must be an integer greater than or equal to zero.
- `LikesCounter` must be an integer greater than or equal to zero.

- [ ] Also please add specs for every validation you will create :+1:

Reply:
- [ ] As mentioned by the previous ☝️ reviewer please add all validations mentioned by microverse but for now, you just add the title and text not be blank so add the following validations as well 🙂

The title must not exceed 250 characters.

CommentsCounter must be an integer greater than or equal to zero.

LikesCounter must be an integer greater than or equal to zero.